crm_claim_rma/models/crm_claim.py

```diff
@@ -0,0 +1,210 @@
+
+    def _get_default_warehouse(self):
+        company_id = self.env.user.company_id.id
+        wh_obj = self.env['stock.warehouse']
+        wh = wh_obj.search([('company_id', '=', company_id)], limit=1)
+        if not wh:
+            raise exceptions.UserError(
+                _('There is no warehouse for the current user\'s company.')
+            )
+        return wh
+
+    def _get_picking_ids(self):
+        """ Search all stock_picking associated with this claim.
+
+        Either directly with claim_id in stock_picking or through a
+        procurement_group.
+        """
+        picking_model = self.env['stock.picking']
+        for claim in self:
+            claim.picking_ids = picking_model.search([
+                '|',
+                ('claim_id', '=', claim.id),
+                ('group_id.claim_id', '=', claim.id)
+            ])
+
+    @api.multi
+    def name_get(self):
+        res = []
+        for claim in self:
+            code = claim.code and str(claim.code) or ''
+            res.append((claim.id, '[' + code + '] ' + claim.name))
+        return res
+
+    company_id = fields.Many2one(change_default=True,
+                                 default=lambda self:
+                                 self.env['res.company']._company_default_get(
+                                     'crm.claim'))
+
+    claim_line_ids = fields.One2many('claim.line', 'claim_id',
+                                     string='Return lines')
+    planned_revenue = fields.Float('Expected revenue')
+    planned_cost = fields.Float('Expected cost')
+    real_revenue = fields.Float()
+    real_cost = fields.Float()
+    invoice_ids = fields.One2many('account.invoice', 'claim_id', 'Refunds',
+                                  copy=False)
+    picking_ids = fields.One2many('stock.picking',
+                                  compute=_get_picking_ids,
+                                  string='RMA',
+                                  copy=False)
+    invoice_id = fields.Many2one('account.invoice', string='Invoice',
+                                 help='Related original Cusotmer invoice')
+    pick = fields.Boolean('Pick the product in the store')
+    delivery_address_id = fields.Many2one('res.partner',
+                                          string='Partner delivery address',
+                                          help="This address will be used to "
+                                          "deliver repaired or replacement "
+                                          "products.")
+    sequence = fields.Integer(default=lambda *args: 1)
+    warehouse_id = fields.Many2one('stock.warehouse', string='Warehouse',
+                                   required=True,
+                                   default=_get_default_warehouse)
+    rma_number = fields.Char(size=128, help='RMA Number provided by supplier')
+
+    @api.model
+    def _get_claim_type_default(self):
+        claim_type = self.env['crm.claim.type'].search([])
```

Comment:
@mdietrichc2c If None is found, it means that crm_claim_type.crm_claim_type_customer does not exists too... So it should never append as the module installs types.

Maybe it should return crm_claim_type_customer directly ?